README.md

```diff
@@ -17,22 +17,36 @@ deploy changes with:
 Editing
 ------------------------------
 
-### How to set up the project
+### Prerequisites
 
-visit: https://github.com/lord/slate
+You're going to need:
+
+ - **Linux or macOS** — Windows may work, but is unsupported.
+ - **Ruby, version 2.3.1 or newer**
+ - **Bundler** — If Ruby is already installed, but the `bundle` command doesn't work, just run `gem install bundler` in a terminal.
 
 ### Getting Set Up
 
-run locally:
+1. Fork this repository on GitHub.
+2. Clone *your forked repository* (not our original one) to your hard drive with `git clone https://github.com/YOURUSERNAME/slate.git`
+3. `cd slate`
+4. Initialize and start Slate. You can either do this locally, or with Vagrant:
 
 ```shell
-./run.sh
+# either run this to run locally
```

Comment:
but we want to use `./run.sh` to execute it right ?

Reply:
It is already at the top of readme.md file. This was changed by accident.